Runtime/Export/AttributeHelperEngine.cs

```diff
@@ -26,11 +26,17 @@ namespace UnityEngine
 {
     class AttributeHelperEngine
     {
+        [System.NonSerialized] static Dictionary<Type, Type> mBaseTypeLookup = new Dictionary<Type, Type>();
```

Comment:
Unity field member prefix coding style is with `m_`.

Reply:
Even in 2018?! Pfff...